### DISCUSSION TITLE
Release 1.7.3 doesn't include mentioned Windows changes

### DISCUSSION THREAD

**Discussion by Vekhir** [OpenBoard-org/OpenBoard#1190] [General]
@kaamui I don't use Windows myself, but I noticed in the Release Notes:

> fixed an issue where the Windows package would not contain the good version of openssl dlls

I guess this refers to fb3f3db85e5467079e54dd8a474df230cfc584dd, but that one is only on `dev`, not on `master` and it's also not in the source tarball for 1.7.3.

If it's something else, please ignore this, but I thought to mention it in case it's a genuine mistake.

**Comment by kaamui** [comment]
Hi,

Thanks you for the heads up. It's a mistake ! I'm not sure why the commit was not merged. I'll fix it before making the Windows package, thank you.
